src/SFML/Graphics/Font.cpp

```diff
@@ -375,30 +376,37 @@ bool Font::hasGlyph(Uint32 codePoint) const
 
 
 ////////////////////////////////////////////////////////////
-float Font::getKerning(Uint32 first, Uint32 second, unsigned int characterSize) const
+float Font::getKerning(Uint32 first, Uint32 second, unsigned int characterSize, bool bold) const
 {
     // Special case where first or second is 0 (null character)
     if (first == 0 || second == 0)
         return 0.f;
 
     FT_Face face = static_cast<FT_Face>(m_face);
 
-    if (face && FT_HAS_KERNING(face) && setCurrentSize(characterSize))
+    if (face && setCurrentSize(characterSize))
```

Comment:
Since this block now handles the "no kerning" case as well, the comment in the "else" block should be adjusted.

Reply:
Shall I just leave `invalid font` or provide something more meaningful?

Reply:
Whatever. Even removing the comment would be fine.